_includes/components/search-results/SearchResult/partials/createLinkText.js

```diff
@@ -0,0 +1,7 @@
+export default function (innerTabKey) {
+  switch (innerTabKey) {
```

Comment:
I'd recommend using a variable called sphere at this point - something more semantic and less tied to the UI. That'd force that variable to be scoped appropriately and avoid weird things happening when we add a tab that isn't a sphere. Just a tip, not a change request for this PR

Reply:
Good suggestion. Will change in CSO dataset branch.